ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -589,6 +589,57 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: aws-ipi-ovn-hypershift-guest-p1-f7
```

Comment:
When a profile is both covered on amd and arm, migrate half of the tests from amd to arm.
E.g, for a f7 profile which only covers on amd previously, and now covers on both amd and arm, use f14 for both of amd and arm.

Reply:
Currently, this is the first ARM profile. Content is Control plane on ARM64, Hosted cluster on AMD64. In 4.14, we may have  Content is Control plane on AMD64, Hosted cluster on ARM64. Then we will consider to use f14 for both profiles.

Reply:
Isn't it an earlier agreement that migrate half of the test from amd to arm when adding new profiles for arm ?

Reply:
Yes, this is agreement. But currently, on hypershift cluster installation, this is the only profile. Rest of ARM or AMD profiles are using for OCP standard cluster.
In additional, no worry about future profile adding. There won't be much ARM profiles on HyperShift side. 
Based on roadmap, we may have 
AWS:
1. Control Plane: ARM64 + Hosted cluster: AMD64;
2. Control Plane: AMD64 + Hosted cluster: ARM64;
Azure:
1. Control Plane: ARM64 + Hosted cluster: AMD64;
2. Control Plane: AMD64 + Hosted cluster: ARM64;